website/docs/d/is_bare_metal_server_network_interface.markdown

```diff
@@ -57,7 +57,14 @@ In addition to the argument reference list, you can access the following attribu
 	- `name` - (String) The user-defined or system-provided name for this reserved IP
   - `resource_type` - (String)The resource type [ **subnet_reserved_ip** ]
 - `security_groups` - (Array) Collection of security groups
-- `status` - (String) The status of the network interface. Supported values are [ **available**, **deleting**, **failed**, **pending** ]
+- `status` - (String) The status of the network interface. 
+
+  ->**Supported Statuses:**
```

Comment:
Can we follow some constant naming i see ->**Supported Statuses:** and some  -> **Supported Status** 
I think  -> **Supported Status**  looks good

Reply:
resolved, and changes are updated.